network-api/networkapi/management/commands/update_site_domain.py

```diff
@@ -1,11 +1,11 @@
-from django.core.management.base import BaseCommand
-from django.contrib.sites.models import Site
-
-
-class Command(BaseCommand):
-    help = 'update the default django site domain to localhost:8000'
-
-    def handle(self, *args, **options):
-        site = Site.objects.get(id=1)
-        site.domain = 'localhost:8000'
-        site.save()
+from django.core.management.base import BaseCommand
```

Comment:
I think this is because I did a global CR+LF => LF line ending replacement... were CR+LF crucial here?